internal/config/config.go

```diff
@@ -319,9 +325,35 @@ func peerFromCR(p metallbv1beta1.BGPPeer) (*Peer, error) {
 		}
 	}
 
+	if p.Spec.Password != "" && p.Spec.SecretRef != nil {
+		return nil, fmt.Errorf("can not have both password and secret ref set in the peer config")
```

Comment:
Return the peer name, it's gonna be easier for the user to understand which bgppeer is wrong

Reply:
done